src/chapters/functions/studio.rst

```diff
@@ -0,0 +1,169 @@
+Studio: Shapes
```

Comment:
Rename: "Studio: Functions" to be consistent with the other chapters.

This is a nice stroll down memory lane.  :-)

Pyyyyyyythoooooooonnnnnnnn!